test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -451,5 +477,92 @@ private static void DeleteDirectory(string path)
 
             }
         }
+
+        //temporary added methods for processing deps.json files for patching
+        private void TempPatching(string patchDir)
+        {
+            var prefix = patchDir;
+            string[] filenames = new string[3] { "dotnet.deps.json", "MSBuild.deps.json", "NuGet.CommandLine.XPlat.deps.json" };
+            string[] fullnames = new string[3];
+            for(int i = 0; i < filenames.Length; i++) 
+            {
+                fullnames[i] = prefix + Path.DirectorySeparatorChar + filenames[i];
+            }
+            TempPatchingDepsJsonForNewlyAddedDlls(fullnames);
+
+        }
+        private void TempCopyNewlyAddedDlls(string[] dlls, string pathCopyFrom, string PathCopyTo)
+        {
+            foreach (var dll in dlls) 
+            {
+                var copyFromFullName = Path.Combine(pathCopyFrom, dll);
+                var copyToFullName = Path.Combine(PathCopyTo, dll);
+                File.Copy(copyFromFullName, copyToFullName);
+            }
+        }
+        private void TempPatchingDepsJsonForNewlyAddedDlls(string[] filePaths)
```

Comment:
`filePaths` is a great parameter name.  It tells you that it is a collection of file paths.  I'd expect `file` to represent a file (e.g.:  `FileInfo`), `Dir` to represent a directory (e.g.:  `DirectoryInfo`), and `dll` maybe to represent a .NET assembly (e.g.:  `Assembly`).

With that in mind rename:
- [ ] [`patchDir`](https://github.com/NuGet/NuGet.Client/pull/3165/files#diff-3a22160d75af0ed49d60562b6157b371R482) to `patchDirectoryPath`
- [ ] [`dlls`](https://github.com/NuGet/NuGet.Client/pull/3165/files#diff-3a22160d75af0ed49d60562b6157b371R494) to `fileNames`
- [ ] [`pathCopyFrom`](https://github.com/NuGet/NuGet.Client/pull/3165/files#diff-3a22160d75af0ed49d60562b6157b371R494) to `copyFromPath`
- [ ] [`PathCopyTo`](https://github.com/NuGet/NuGet.Client/pull/3165/files#diff-3a22160d75af0ed49d60562b6157b371R494) to `copyToPath`
- [ ] [`dll`](https://github.com/NuGet/NuGet.Client/pull/3165/files#diff-3a22160d75af0ed49d60562b6157b371R496) to `fileName`
- [ ] [`file`](https://github.com/NuGet/NuGet.Client/pull/3165/files#diff-3a22160d75af0ed49d60562b6157b371R506) to `filePath`